### DISCUSSION TITLE
setup.py

### DISCUSSION THREAD

**Discussion by Itachidbz** [python/typing#1496] [Q&A] [answered]
why there are no setup.py after 3.10.0.2 and what is installation proccess for the versions after that

**Comment by JelleZijlstra** [comment, marked as answer]
I assume you are asking about https://github.com/python/typing_extensions.

typing-extensions is now built using pyproject.toml (https://github.com/python/typing_extensions/blob/main/pyproject.toml), following [PEP 517](https://peps.python.org/pep-0517/). If you have a source tree, you can install from it with e.g. `pip install /path/to/typing-extensions`.
